### PR TITLE
Change in JJB to be able send messages to ci message bus

### PR DESCRIPTION
**Pull request by mkluson**
To be able generate jobs, which send messages to ci message bus,
it is necessary to add config file to get_job_as_xml. The config
is created as a temporary and it contains the jenkins master
URL and credentials (if it is required). Credentials are obtained
from environment variables.

### CODE REVIEW

**Comment by optak** [comment]
LGTM. Conflict is not serious, only mkluson has several lines added, easy to fix.
As I understand it, this patch is blocking baseos-ci from sending test results to message bus. Baseos-ci part should be merge after this is merged.
Can you please @gsr-shanks or @lulinqing merge this soon after @mkluson solve conflict?

**Review by lulinqing — APPROVED** [review]

**Comment by ralphbean** [comment]
:bus:

**Pull request merged**
